src/tsconfig.json

```diff
@@ -9,13 +9,11 @@
         "noImplicitReturns": true,
         "noUnusedLocals": true,
         "noUnusedParameters": false,
-        "removeComments": true,
```

Comment:
Please, keep the top `tsconfig.json` with `removeComments: true` and override it in the `client/tsconfig.json` with `removeComments: false`. By default, we want to keep the safe options at the top level.